### ISSUE TITLE
Add keyboard navigation section under the Grid Sample 

### ISSUE THREAD

**Issue by SlavUI** [IgniteUI/igniteui-docfx#610]
We should add a tab on the right hand navigation in the Grid Sample section describing the Keyboard navigation of the Grid.
 
![capture](https://user-images.githubusercontent.com/33216749/43509430-56beff86-957b-11e8-858f-8db547fdeaca.PNG)

**Comment by zdrawku** [comment]
@ddincheva once we finish with the keyboard navigation updates, we should add a section for this in the main grid topic. This section should be combination between the [wiki ](https://github.com/IgniteUI/igniteui-angular/wiki/igxGrid-Specification#navigation)page and all new enhancements/changes.

**Comment by SlavUI** [comment]
This was added: https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid.html#keyboard-navigation

Closing the issue
